packages/flutter_tools/lib/src/build_system/targets/web.dart

```diff
@@ -631,11 +631,5 @@ function onlineFirst(event) {
     })
   );
 }
-
-function skipWaiting() {
-  if (self.skipWaiting != undefined) {
-    self.skipWaiting();
-  }
-}
```

Comment:
Was this here just for sanity before? Or why did we need to check for `self.skipWaiting` being a thing?

Reply:
yeah, not needed - also infinite loop